licenses.go

```diff
@@ -41,43 +41,42 @@ func SetFirstEnterpriseCommit(sha string) {
 	firstEnterpriseCommit = sha
 }
 
-func CheckMenderCompliance(t *testing.T) error {
+func CheckMenderCompliance(t *testing.T) {
 	pathToTool, err := locatePackage()
 	if err != nil {
-		return err
+		t.Errorf("Failed to locate package: %s", err)
 	}
 
-	cmdString := []string{path.Join(pathToTool, "check_license_go_code.sh")}
+	args := []string{"-c", fmt.Sprintf("source %s", path.Join(pathToTool, "check_license_go_code.sh"))}
```

Comment:
I don't understand this. Why do you want to source the script? By looking at the script it will work, but I think it is a bad practice: `source` should be used upon files containing functions or env variables, not actual bash code.

I though that this would be as simple as calling `bash [script]` (i.e. `bash check_license_go_code.sh`), wouldn't that work?

Reply:
Ur right, that is way easier! Thx

:racehorse: